### ISSUE TITLE
Black background

### ISSUE THREAD

**Issue by nab0y4enko** [SCENEE/FloatingPanel#22]
Hey. Thank you so much for your work. 

In version 1.1, I got a problem.
If I forbid .full state, then when I swipe up, the background turns black. I recorded a video
https://monosnap.com/file/3zniKRgfEHHnCXIYZCc4bSUQsDMNhf

```

extension CurrentFlightViewController: FloatingPanelLayout {

    public var initialPosition: FloatingPanelPosition {
        return .tip
    }
    public var supportedPositions: Set<FloatingPanelPosition> {
        return [.tip, .half]
    }
    
    public func insetFor(position: FloatingPanelPosition) -> CGFloat? {
        switch position {
        case .tip:
            return insuranceViewController.headerHeight
        case .half:
            return insuranceViewController.allContentHeight
        default:
            return 0
        }
    }
    
    public func prepareLayout(surfaceView: UIView, in view: UIView) -> [NSLayoutConstraint] {
        if #available(iOS 11.0, *) {
            return [
                surfaceView.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor, constant: 0.0),
                surfaceView.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor, constant: 0.0)
            ]
        } else {
            return [
                surfaceView.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor, constant: 0.0),
                surfaceView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor, constant: 0.0)
            ]
        }
    }
}
```

Could you help me with this issue? 
Thanks!

**Comment by SCENEE** [comment]
I can reproduce this issue and the above suggestion of mine is wrong. I will fix it.

**Comment by SCENEE** [comment]
@nab0y4enko, 
I fixed this issue on PR #27 😄 
 If you still have the same problem, please reopen this issue.
